testsuite/MDAnalysisTests/coordinates/test_gro2.py

```diff
@@ -0,0 +1,24 @@
+from MDAnalysis.coordinates.GRO import GROReader, GROWriter
+from MDAnalysisTests.coordinates.base import BaseReference, BaseReaderTest
+from MDAnalysisTests.datafiles import COORDINATES_GRO
+import numpy as np
+
+
+class GROReference(BaseReference):
+    def __init__(self):
+        super(GROReference, self).__init__()
+        self.trajectory = COORDINATES_GRO
+        # self.topology = COORDINATES_GRO
+        self.reader = GROReader
+        self.writer = GROWriter
+        self.ext = 'gro'
+        # self.volume = 0
+        # self.dimensions = np.zeros(6)
+        # self.container_format = True
```

Comment:
You need to deactivate all tests that skip frames or otherwise work on the assumption that a gro stores a trajectory instead of a single snapshot. For this you need to change the BaseReaderTest.